insitu/templatetags/tags.py

```diff
@@ -0,0 +1,37 @@
+from django import template
```

Comment:
Use context preprocessors to send the statistics to view. Write one function ( statistics) that returns all the values to template